docs/visualization/plotting-overview.ipynb

```diff
@@ -308,6 +308,58 @@
    "source": [
     "(100 * (da1 - da1.min())).plot(norm='log')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Migrating to the New Backend\n",
+    "\n",
+    "Since version 0.17, Scipp can use either the old builtin plotting backend or [Plopp](https://scipp.github.io/plopp/).\n",
+    "The old backend is deprecated and will be removed in the first release in or after August 2023.\n",
```

Comment:
```suggestion
    "The old backend is deprecated. It will be removed at the earliest in Scipp v23.08.0 (August 2023).\n",
```

Reply:
I wouldn't name a concrete version number because that might never exist. Also, saying 'at the earliest' is quite imprecise and is open to the interpretation that there will be an additional announcement with a concrete date and until then, it is fine to ignore the warning.

Reply:
> Also, saying 'at the earliest' is quite imprecise and is open to the interpretation that there will be an additional announcement with a concrete date and until then, it is fine to ignore the warning.

I don't want to commit to making a release just for removing something, that's why "at the earliest". You can qualify by adding "without further notice", if you like.

Reply:
> I don't want to commit to making a release just for removing something, that's why "at the earliest".

My formulation achieves the same.

> I would name a version numbers, because people can then pin an upper bound (not that we would recommend that, but they could).

So why should we make concessions for that?

Reply:
Please change it. I consider it is important to mention the version number, as not every reader will be familiar with how to map dates to version numbers.